test/Numerics/DGMethods/Euler/acousticwave_1d_imex.jl

```diff
@@ -102,6 +102,7 @@ function run(
     vert_range = grid1d(
         _planet_radius,
         FT(_planet_radius + setup.domain_height),
+        NoStretching(),
```

Comment:
Not needed if you give `stretch` a default parameter as suggested above. And ditto in the other tests... no changes needed at all.

Reply:
Got it, I'll adjust everything as you suggested